### PR TITLE
rename verify/ to hack/ for consistency with k/k

### LINKED ISSUE #5011: Rename /verify directory to /hack or something
There are a couple of scripts in the /verify directory that update stuff like bazel files or gofmt. It would be more intuitive to have a different name than `verify`.

cc: @cjwagner @spxtr

### PR DESCRIPTION
**Pull request by BenTheElder**
fixes #5011, also adds `hack/update-all.sh`

now you can run `hack/update-all.sh` e̶v̶e̶r̶y̶w̶h̶e̶r̶e̶ in kubernetes and test-infra! 😆

### CODE REVIEW

**Comment by BenTheElder** [comment]
/area bikeshed
:joy:

**Comment by BenTheElder** [comment]
These are all failing since they try to run the script from the old location 🤷‍♂️

**Comment by krzyzacy** [comment]
seems now we have `hack/foo.sh` and `hack/verify/foo.sh` now
otherwise it's fine

**Comment by BenTheElder** [comment]
>seems now we have hack/foo.sh and hack/verify/foo.sh now otherwise it's fine

I really don't know how I managed to do that, it should be fixed now.

**Comment by BenTheElder** [comment]
I should add a `hack/update-all.sh` so you can run that in k/k or k/t-i and know your branch is well-formatted :thinking:

**Comment by BenTheElder** [comment]
Added `hack/update-all.sh`.

**Comment by cblecker** [comment]
> /area bikeshed
Can this be an alias for `/shrug`? 😂

**Comment by BenTheElder** [comment]
> Can this be an alias for /shrug? :joy:

SGTM, @krzyzacy :smile: ?

File: hack/update-all.sh [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+TESTINFRA_ROOT=$(git rev-parse --show-toplevel)
+${TESTINFRA_ROOT}/hack/update-bazel.sh
+${TESTINFRA_ROOT}/hack/update-gofmt.sh
```
Comment (kargakis) [inline hack/update-all.sh]: Is `update-gofmt.sh` mutating the actual files or is it sending its output to stdout? If the latter, I am not sure how useful it is. Maybe a follow-up issue to fix it to actually update.

Comment (BenTheElder) [reply hack/update-all.sh]: It's mutating them if they need formatting (it has the -w flag).

**Comment by BenTheElder** [comment]
Anyone? I'm often typing `hack/update-bazel.sh` after switching between k/k and k/t-i [and it seems we need to keep these more in sync](https://github.com/kubernetes/test-infra/pull/5301#discussion_r148429984).

**Comment by krzyzacy** [comment]
/lgtm
/hold
manually merge and watch the 🔥 tomorrow?
`bazel-test-affected.sh` might already be a dead script

**Comment by cblecker** [comment]
This conflicts with #5301.. so one has to go first and the other rebase. I don’t have a preference as to which :)

**Comment by BenTheElder** [comment]
Let's get the dep fix in first then I can resume changing the shed color :-)

**Comment by BenTheElder** [comment]
should be good to go now :bike: :derelict_house: :fire:
/hold cancel

**Comment by BenTheElder** [comment]
missed the test-infra bazel jobs specifying `//verify:verify-all` somehow (bad rebase?), fixed in the follow-up.

**Comment by stevekuznetsov** [comment]
Would have been nice to symlink or e-mail blast this one :)

**Pull request merged**